openapi/README.md

```diff
@@ -8,7 +8,7 @@ The list of supported paths should be ajusted in `ogcapi-processes-3.yaml`.
 
 The `ogcapi-processes-3.bundled.json` was generated with `swagger-cli bundle` from `ogcapi-processes-3.yaml` and its dependencies included from the components sub-directories.
 
-The `paths/processes-dru/pProcessListDeploy.yaml` and `paths/processes-dru/pProcessDescriptionReplaceUndeploy.yaml` files are generated by concatening multiple files in a single one. you can sue the command bellow to update their value with the current schema:
+The `paths/processes-dru/pProcessListDeploy.yaml` and `paths/processes-dru/pProcessDescriptionReplaceUndeploy.yaml` files are generated by concatening multiple files in a single one. you can use the command bellow to update their value with the current schema:
```

Comment:
Thanks @gfenoy . _bellow_ is still wrong though :)

Reply:
Thanks for you patience @jerstlouis

Reply:
@gfenoy No thank you for yours ;) Sorry for being so annoying finding things to fix :P